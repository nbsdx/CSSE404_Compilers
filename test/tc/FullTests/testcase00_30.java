/* Invalid: Method Overloading */
class Foo 
{ 
	public static void main(String[] args) 
	{ 
	}
}

class Bar 
{ 
	public Bar self() 
	{ 
		return this; 
	} 
}

class Baz extends Bar 
{ 
	public Bar self(int a) 
	{ 
		return this; 
	} 
}