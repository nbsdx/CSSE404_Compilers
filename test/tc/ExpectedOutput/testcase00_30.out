Cannot overload methods.  Method self has different type signature than inherited method of the same name.
